pkg/cmd/oauth-apiserver/cmd.go

```diff
@@ -139,6 +144,8 @@ func (o *OAuthAPIServerOptions) NewOAuthAPIServerConfig() (*apiserver.Config, er
 		return nil, err
 	}
 
+	// we disable PriorityAndFairness for extension servers
+	serverConfig.GenericConfig.FlowControl = nil
```

Comment:
this works but it is ugly, better if we could control it via a command line flag passed from an operator.